lib/stripe/singleton_api_resource.rb

```diff
@@ -6,7 +6,9 @@ def self.resource_url
       if self == SingletonAPIResource
         raise NotImplementedError, "SingletonAPIResource is an abstract class.  You should perform actions on its subclasses (Account, etc.)"
       end
-      "/v1/#{CGI.escape(class_name.downcase)}"
+      # Namespaces are separated in object names with periods (.) and in URLs
+      # with forward slashes (/), so replace the former with the latter.
+      "/v1/#{self::OBJECT_NAME.downcase.tr('.', '/')}"
```

Comment:
A bit disappointing that we have the same code and comment twice. Is there a way to share the code? :)

Reply:
Haha, you mean from stripe-php?

Maybe once all these languages are running on the LLVM! ;)